requirements.txt

```
@@ -8,8 +8,8 @@ peft>=0.12.0
 decord>=0.6.0
 wandb
 pandas
-torch>=2.4.0
-torchvision>=0.19.0
+torch>=2.5.0
+torchvision>=0.20.0
```

```suggestion
torchvision<0.20.0
```

Same reason as above